WTWSMS/localisation/WtWSMS_nicknames.csv

```diff
@@ -24,7 +24,7 @@ nick_dicoros;Dicoros;Dicoros;;;;;;;;;;;;x
 nick_coscrach;Coscrach;Coscrach;;;;;;;;;;;;x
 nick_lonn;Lonn;Lonn;;;;;;;;;;;;x
 nick_dub;Dub;Dub;;;;;;;;;;;;x
-nick_the_dismal;the Dismal;;;;;;;;;;;;;x
+nick_simhavikrantagami;Simhavikrantagami;;;;;;;;;;;;;x
```

Comment:
Was it intentional that you removed the other localisation entry?

Reply:
Yes, nick_the_dismal is already localised

Reply:
Oh, OK.